flink-runtime/src/main/java/org/apache/flink/runtime/deployment/CachedShuffleDescriptors.java

```diff
@@ -104,4 +122,14 @@ public void markPartitionFinished(IntermediateResultPartition resultPartition) {
                         checkNotNull(
                                 resultPartitionIdToIndex.get(resultPartition.getPartitionId()))));
     }
+
+    private boolean isForceOffload(ShuffleDescriptorAndIndex[] shuffleDescriptorsToSerialize) {
```

Comment:
How about `shouldOffload` instead of `isForceOffload`? I think we're not forced to do this optimization.🤣